comma.nix

```diff
@@ -0,0 +1,60 @@
+{ pkgs
+, stdenv
+, lib
+, fetchurl
+, nix-index
+, nix
```

Comment:
This is currently still nix 2.3 on 21.11, which itself does not officially support flakes. So I would recommend to use `nixFlakes` here instead. Otherwise the older nix will complain about `warning: unknown setting 'experimental-features'`, which is how you enable flake support in the first place.